docker-entrypoint.sh

```diff
@@ -16,8 +16,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+export NGIX_PATH=/usr/share/nginx/html
```

Comment:
typo in the variable name?

```suggestion
export NGINX_PATH=/usr/share/nginx/html
```